Round Robin (#210)
jaybee84/ml-in-rd: content/06.model-complexity.md

## Reducing misinterpretation of model output with statistical techniques 

> Machine learning methods generally work well on data that meets a few critical assumptions. 

Author: ```suggestion
Machine learning methods generally work well on data that meet a few critical assumptions. 
```

For example, LASSO has been used to capturing combinations of rare and common variants associated with specific traits have proven beneficial [@doi:10.1186/1753-6561-5-s9-s113].
In this example, applying LASSO regularization reduced the number of common variants included as features in the final analysis generating a simpler model while reducing error in the association of common and rare variants with a specific trait. 
In the context of rare immune cell signature discovery, variations of elastic-net regression were found to outperform other regression approaches [@doi:10.1111/j.1467-9868.2005.00503.x; @doi:10.1016/j.compbiomed.2015.10.008; @doi:10.1186/s12859-019-2994-z]. 
Thus, regularization methods like LASSO or elastic-net are beneficial in ML with rare observations, and are worth exploring in the context of rare diseases.[@doi:10.1371/journal.pgen.1004754] 
Other examples of regularization that have been successfully applied to rare disease ML include Kullback–Leibler divergence (KL divergence) and dropout regularization.

> In a study using a variational auto encoder (VAE) for dimensionality reduction in gene expression data from AML samples, the KL divergence between the input data and its low dimensional representation provided the regularizing penalty for the model.[@doi:10.1101/278739; @doi:10.48550/arXiv.1312.6114]

Reviewer: Need to define AML now that the sentence before this one no longer does.

Author: ```suggestion
In a study using a variational auto encoder (VAE) for dimensionality reduction in gene expression data from acute myeloid leukemia (AML) samples, the KL divergence between the input data and its low dimensional representation provided the regularizing penalty for the model.[@doi:10.1101/278739; @doi:10.48550/arXiv.1312.6114]
```

> In a study using a convolutional neural network (CNN) to identify tubers in MRI images from TS patients, overfitting was minimized using the dropout regularization method which removed randomly chosen network nodes in each iteration of the CNN model generating simpler models in each iteration.[@doi:10.1371/journal.pone.0232376]

Reviewer: Similarly

```suggestion
In a study using a convolutional neural network (CNN) to identify tubers in MRI images from tuberous sclerosis patients, overfitting was minimized using the dropout regularization method which removed randomly chosen network nodes in each iteration of the CNN model generating simpler models in each iteration.[@doi:10.1371/journal.pone.0232376]
```
